index.js

```diff
@@ -1,45 +1,14 @@
-// nr = not as per eslint:recommended
+import baseConfig from './base-config.js';
+import browserConfig from './browser-config.js';
+import litConfig from './lit-config.js';
+import nodeConfig from './node-config.js';
+import reactConfig from './react-config.js';
+import testingConfig from './testing-config.js';
```

Comment:
You should be able to do this in 1 line:

```javascript
export { testingConfig as testing } from './testing-config.js';
```

I'd also be OK with just keeping the name as `testingConfig`, etc. 🤷